src/js/views/patients/worklist/worklist_views.js

```diff
@@ -222,6 +223,43 @@ const EmptyFindInListView = View.extend({
   `,
 });
 
+const CountView = View.extend({
+  template: hbs`
+    {{#if hasReachedMaximum}}
+      <div>
+        {{#if isFindInListApplied}}
+          {{#if isFlowList}}
+            {{formatMessage (intlGet "patients.worklist.worklistViews.countView.maximumFlowsCountNarrowed") itemCount=count maximumCount=maximumCount}}
+          {{else}}
+            {{formatMessage (intlGet "patients.worklist.worklistViews.countView.maximumActionsCountNarrowed") itemCount=count maximumCount=maximumCount}}
+          {{/if}}
+        {{else}}
+          {{formatMessage (intlGet "patients.worklist.worklistViews.countView.maximumListCount") maximumCount=maximumCount listTypeLabel=listTypeLabel}}
+        {{/if}}
+      </div>
+      <div>{{ @intl.patients.worklist.worklistViews.countView.narrowFilters }}</div>
+    {{else}}
+      <span class="worklist-list__count-total">
+        {{#if isFlowList}}
+          {{formatMessage (intlGet "patients.worklist.worklistViews.countView.flowsCount") itemCount=count}}
+        {{else}}
+          {{formatMessage (intlGet "patients.worklist.worklistViews.countView.actionsCount") itemCount=count}}
+        {{/if}}
+      </span>
+    {{/if}}
+  `,
+  templateContext() {
+    return {
+      maximumCount: this.getOption('maximumCount'),
+      hasReachedMaximum: this.getOption('hasReachedMaximum'),
+      count: this.getOption('count'),
```

Comment:
Do we want to add thousand separator formatting to the `count` and `maximumCount` (i.e. `1000` => `1,000`? 

It's not a problem now. But in the future, the capped results could be more than 1,000?

Could just throw a `.toLocaleString()` on it.

Just a thought as I was reviewing this.

Reply:
I don't think we need to worry about the separator for now.